src/main/resources/application.properties

```diff
@@ -0,0 +1,83 @@
+#####-----------------------------------------------######
+### ---------[  GENERAL Configurations   ]---------###
+#####-----------------------------------------------######
+#spring.application.name=fruits-ecommerce
+server.port=9090
+spring.mustache.check-template-location=false
+### -  development-environment
+spring.jpa.open-in-view=true
+server.error.include-message=always
+### -   Production-environment
+#spring.jpa.open-in-view=false
+#server.error.include-message=never
+
+#####-----------------------------------------------######
+### ----- [  Data-base Configurations ]--------####
+#####-----------------------------------------------######
+spring.datasource.url=jdbc:mysql://localhost:3306/fruit_ecommerce?allowPublicKeyRetrieval=true&useSSL=false&serverTimezone=UTC
+#spring.datasource.url=jdbc:mysql://localhost:3306/fruit_ecommerce?allowPublicKeyRetrieval=true
+spring.datasource.username=fruit_ecommerce
+spring.datasource.password=fruit_ecommerce
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+
+#####-----------------------------------------------######
+###- JPA- Configuration #
+#####-----------------------------------------------######
+spring.jpa.show-sql=true
+spring.jpa.hibernate.ddl-auto=none
+#spring.jpa.hibernate.ddl-auto=validate
```

Comment:
Let's remove this commented block here

Reply:
Well done, I will work on implementing it for the better as you indicated.